encrypt/encrypt.go

```diff
@@ -66,13 +66,29 @@ func Encrypt(args []string) error {
 	files := []helpers.EncryptionFileSet{}
 	for _, filename := range Args.Args() {
 
+		// Input filename is not found
+		if !helpers.FileExists(filename) {
+			return fmt.Errorf("Input filename is not found")
+		}
+
+		// Check if the input filename is already encrypted
+		fileExtension := filepath.Ext(filename)
+		if fileExtension == ".c4gh" {
+			return fmt.Errorf("Input filename is already encrypted(.c4gh)")
+		}
```

Comment:
This doesn't verify that the file is encrypted, only that it has a `.c4gh` file ending, which by it self is not enough.
Crypt4gh files have a `magic number` at the beginning of the header that will tell you that the file is encrypted.